images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -487,6 +487,11 @@ function Get-PowershellVersion {
     return $powershellVersion
 }
 
+function Get-SwigVersion {
+    $swigVersion = Run-Command "swig -version" | Take-Part -Part 3
```

Comment:
This is the output of this command, output is incorrect
![image](https://user-images.githubusercontent.com/8777209/120770894-4d86ac00-c527-11eb-8427-84180e0100d8.png)

Reply:
ok

Reply:
fixed

Reply:
> This is the output of this command, output is incorrect
> ![image](https://user-images.githubusercontent.com/8777209/120770894-4d86ac00-c527-11eb-8427-84180e0100d8.png)

fixed

Reply:
> This is the output of this command, output is incorrect
> ![image](https://user-images.githubusercontent.com/8777209/120770894-4d86ac00-c527-11eb-8427-84180e0100d8.png)

fixed